### PR TITLE
Test_script/ rake task added to test code run api response time

### PR DESCRIPTION
**Pull request by somnath77**
https://trello.com/c/QcvGWqoc

### CODE REVIEW

File: lib/tasks/test_script.rake [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# RAILS_ENV=production bundle exec rake test_script:code_run[:drive_id,:sleep_period]
+namespace :test_script do
```
Comment (anilmaurya) [inline lib/tasks/test_script.rake:4]: rename to `load_testing`

File: lib/tasks/test_script.rake [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  task :code_run, %i[drive_id sleep_period] => [:environment] do |_task, args|
```
Comment (anilmaurya) [inline lib/tasks/test_script.rake:5]: code_run => `run`

File: lib/tasks/test_script.rake [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    thread_array = []
+
+    Drive.find(args[:drive_id]).drives_candidates.each do |dc|
+      dc.submissions.each do |sub|
+        thread = Thread.new do
+          sleep(rand(args[:sleep_period].to_i) / 10.0)
+          parameter = { source_code: sub.answer, language_id: sub.lang_code, stdin: 'hi',
+                        callback_url: "#{ENV['SERVER_URL']}/executions/submission_result?room=dfgrjhwgejrwev" }
```
Comment (anilmaurya) [inline lib/tasks/test_script.rake:13]: remove callback url

File: lib/tasks/test_script.rake (hunk 1 above)
Comment (somnath77) [reply lib/tasks/test_script.rake]: done

File: lib/tasks/test_script.rake (hunk 3 above)
Comment (somnath77) [reply lib/tasks/test_script.rake]: done

File: lib/tasks/test_script.rake (hunk 2 above)
Comment (somnath77) [reply lib/tasks/test_script.rake]: done

**Review by onkar-josh — APPROVED** [review]

**Pull request merged**